src/actions/index.js

```diff
@@ -0,0 +1,4 @@
+export const showMovies = (movies) => ({
```

Comment:
i saw something that said we can also make fetch calls in our actions file. maybe we can look into that later